content/docs/hooks-intro.md

```diff
@@ -25,87 +25,89 @@ function Example() {
 }
 ```
 
-This new function `useState` is the first "Hook" we'll learn about, but this example is just a teaser. Don't worry if it doesn't make sense yet!
+`useState` फंक्शन पहला "हुक" है जिसके बारे में हम सीखेंगे। याद रखिये ये एक्जाम्पल सिर्फ एक टीज़र है। अगर आपको लगे इसका कोई मतलब नहीं बन रहा तो चिंता मत करिये!
 
-**You can start learning Hooks [on the next page](/docs/hooks-overview.html).** On this page, we'll continue by explaining why we're adding Hooks to React and how they can help you write great applications.
+**आप [अगले पेज पर](/docs/hooks-overview.html) हुक्स के बारे में सीखना शुरू कर सकते हैं।** इस पेज पर हम ये बताएँगे की हम रियेक्ट में हुक्स को क्यों शामिल कर रहे हैं और कैसे आप अपनी उनकी सहायता से बेहतरीन एप्लीकेशंस बना सकते हैं।
 
->Note
+
+>नोट
 >
->React 16.8.0 is the first release to support Hooks. When upgrading, don't forget to update all packages, including React DOM.
->React Native supports Hooks since [the 0.59 release of React Native](https://facebook.github.io/react-native/blog/2019/03/12/releasing-react-native-059).
+> React 16.8.0 हुक्स को सपोर्ट करने वाला पहला वर्जन है| अपग्रेड करते समय सभी पैकेजेस (React Dom भी) को अपडेट करना मत भूले।
+> React Native [वर्जन 0.59](https://facebook.github.io/react-native/blog/2019/03/12/releasing-react-native-059) से ही हुक्स को सपोर्ट करता है।
 
-## Video Introduction {#video-introduction}
+##  वीडियो से परिचय  {#video-introduction}
 
-At React Conf 2018, Sophie Alpert and Dan Abramov introduced Hooks, followed by Ryan Florence demonstrating how to refactor an application to use them. Watch the video here:
+React कांफ्रेंस 2018 में सोफी एल्पर्ट और डैन एब्रामोव में रियेक्ट हुक्स को सबसे अवगत करवाया जिसके बाद रायन फ्लोरेंस ने दिखाया की कैसे हम एक एप्लीकेशन को हुक्स की सहायता से रेफैक्टर कर सकते हैं। ये रहा उसका वीडियो:
```

Comment:
`कांफ्रेंस` => `Conf`

this is the name of the conference, we should not translate it